docs/api/devtime_dependencies.md

```
@@ -0,0 +1,275 @@
+	if err != nil {
+		return err
+	}
+
+	s.ctr = c
+	return nil
+}
+
+// String returns a human-readable representation of the dependency's state.
+// It implements the fiber.RuntimeDependency interface.
+func (s *redisStore) String() string {
+	return "redis-store"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)